src/cf_create.cpp

```diff
@@ -168,10 +168,13 @@ void cfedit_parse(struct descriptor_data *d, const char *arg)
     break;
   case CFEDIT_RATING:
     if (option_n > GET_SKILL(CH, SKILL_COMPUTER)) {
-      send_to_char(CH, "You can't create a program of a higher rating than your computer skill.\r\n"
+      send_to_char(CH, "You can't create a complex form of a higher rating than your computer skill.\r\n"
                    "Enter Rating: ");
     } else if (option_n > GET_OTAKU_MPCP(CH)) {
-      send_to_char(CH, "You can't create a program of a higher rating than your living persona's MPCP rating.\r\n"
+      send_to_char(CH, "You can't create a complex form of a higher rating than your living persona's MPCP rating.\r\n"
+                   "Enter Rating: ");
+    } else if (option_n <= 0) {
+      send_to_char(CH, "You can't create a complex form less than 0, you silly bastard.\r\n"
```

Comment:
Funny, but please drop the ad-hom and give it a dispassionate `You can't create a complex form less than 0` for both of these. Game culture can be really influenced by the code, so if the game starts knocking people...